src/core/ext/README.md

```diff
@@ -0,0 +1,5 @@
+Optional plugins for gRPC Core: Modules in this directory extend gRPC Core in
+useful ways.
+
+NOTE: The movement of code between lib and ext is an ongoing effort, so this
```

Comment:
Do we have a plan to finish this up, or at least a list of which code should ideally live in which place?  I have to say that the intended division has always been a bit unclear to me, but if I had a clearer idea of how things were *supposed* to be laid out, I could at least try to move in the right direction over time.